asyncio_mqtt/client.py

```diff
@@ -238,6 +238,10 @@ def _pending_call(self, mid, value):
             self._pending_calls.pop(mid, None)
 
     def _on_connect(self, client, userdata, flags, rc, properties=None):
+        # if alread connected return
```

Comment:
Looks good overall. :+1: Can we make the comment a bit more descriptive? Something like:
```python
# Return early if already connected. Sometimes, paho-mqtt calls _on_connect
# multiple times. Maybe because we receive multiple CONNACK messages
# from the server. In any case, we return early so that we don't set
# self._connected twice (as it raises an asyncio.InvalidStateError).
```